src/ray/core_worker/task_execution.cc

```diff
@@ -21,6 +22,11 @@ CoreWorkerTaskExecutionInterface::CoreWorkerTaskExecutionInterface(
       TaskTransportType::RAYLET,
       std::unique_ptr<CoreWorkerRayletTaskReceiver>(new CoreWorkerRayletTaskReceiver(
           raylet_client, object_interface_, main_service_, worker_server_, func)));
+  task_receivers_.emplace(
+      TaskTransportType::DIRECT_ACTOR,
+      std::unique_ptr<CoreWorkerDirectActorTaskReceiver>(
+          new CoreWorkerDirectActorTaskReceiver(object_interface_, main_service_,
```

Comment:
Should this be created only if direct call is enabled?

Reply:
I thought about this too, but grpc server requires all the grpc services be registered before starting the server, thus this isn't possible..